include/mscclpp/gpu.hpp

```diff
@@ -6,6 +6,8 @@
 
 #if defined(__HIP_PLATFORM_AMD__)
 
+// #include <hip/hip_bf16.h>
```

Comment:
does `<hip/hip_bf16.h>` work for latest rocm?

Reply:
Let me include this again, thanks